relint-pre-commit.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
I don't see file attributes on GitHub, but can you make sure this file is executable (`chmod +x`).